js/flightlog_fields_presenter.js

```diff
@@ -1032,6 +1032,90 @@ function FlightLogFieldPresenter() {
+                    'debug[2]': 'Current PID Setpoint',
+                    'debug[3]': 'Angle Current',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.CURRENT_ANGLE = {
+                    'debug[all]': 'Current Angle',
+                    'debug[0]': 'Current Angle X',
+                    'debug[1]': 'Current Angle Y',
+                    'debug[2]': 'Current Angle Z',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.DSHOT_TELEMETRY_COUNTS = {
+                    'debug[all]': 'DShot Telemetry Counts',
+                    'debug[0]': 'DShot Telemetry Debug[0] + 1',
+                    'debug[1]': 'DShot Telemetry Debug[1] + 1',
+                    'debug[2]': 'DShot Telemetry Debug[2] + 1',
+                    'debug[3]': 'Preamble Skip',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.RPM_LIMIT = {
+                    'debug[all]': 'RPM Limit',
+                    'debug[0]': 'Average RPM',
+                    'debug[1]': 'Average RPM (unsmoothed)',
+                    'debug[2]': 'RPM Limit throttle scale',
+                    'debug[3]': 'Throttle',
+                    'debug[4]': 'Error',
+                    'debug[5]': 'Proportional',
+                    'debug[6]': 'Integral',
+                    'debug[7]': 'Derivative',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.RC_STATS = {
+                    'debug[all]': 'RC Stats',
+                    'debug[0]': 'Average Throttle',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.MAG_CALIB = {
+                    'debug[all]': 'Mag Calibration',
+                    'debug[0]': 'Mag X',
+                    'debug[1]': 'Mag Y',
+                    'debug[2]': 'Mag Z',
+                    'debug[3]': 'Norm / Length of magADC',
+                    'debug[4]': 'Estimated Mag Bias X',
+                    'debug[5]': 'Estimated Mag Bias Y',
+                    'debug[6]': 'Estimated Mag Bias Z',
+                    'debug[7]': 'Mag Bias Estimator',
+                };
+                DEBUG_FRIENDLY_FIELD_NAMES.MAG_TASK_RATE = {
+                    'debug[all]': 'Mag Task Rate',
+                    'debug[0]': 'Task Rate (Hz)',
+                    'debug[1]': 'Actual Data Rate (Hz)',
+                    'debug[2]': 'Data Interval (Us)',
+                    'debug[3]': 'Execute Time (Us)',
+                    'debug[4]': 'Bus Busy',
+                    'debug[5]': 'Read State',
+                    'debug[6]': 'Task Time (Us)',
+                };
+            }
+
+            if (semver.gte(firmwareVersion, '4.5.0')) {
+                DEBUG_FRIENDLY_FIELD_NAMES.FAILSAFE = {
+                    'debug[all]': 'Failsafe',
+                    'debug[0]': 'Failsafe Phase switch',
+                    'debug[1]': 'Failsafe State',
+                    'debug[2]': 'Receiving data from Rx',
+                    'debug[3]': 'Failsafe Phase',
+                };
+            }
+
```

Comment:
These are new debugs.  They should be included in the initial debug list, not here.  Conditional server checks are only required if an existing debug has field names changed in a later firmware version.